staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/controller.go

```diff
@@ -187,30 +200,52 @@ func (c *celAdmissionController) Validate(
 	c.mutex.RLock()
 	defer c.mutex.RUnlock()
 
-	var allDecisions []PolicyDecisionWithMetadata = nil
+	var deniedDecisions []policyDecisionWithMetadata
+
+	addConfigError := func(err error, definition *v1alpha1.ValidatingAdmissionPolicy, binding *v1alpha1.ValidatingAdmissionPolicyBinding) {
+		// we always default the FailurePolicy if it is unset and validate it in API level
+		var policy v1alpha1.FailurePolicyType
+		if definition.Spec.FailurePolicy == nil {
+			policy = v1alpha1.Fail
+		} else {
+			policy = *definition.Spec.FailurePolicy
+		}
 
-	addConfigError := func(err error, definition PolicyDefinition, binding PolicyBinding) {
-		wrappedError := fmt.Errorf("configuration error: %w", err)
-		switch p := definition.GetFailurePolicy(); p {
-		case Ignore:
-			klog.Info(wrappedError)
+		// apply FailurePolicy specified in ValidatingAdmissionPolicy, the default would be Fail
+		switch policy {
+		case v1alpha1.Ignore:
+			if binding == nil {
+				klog.Infof("ignored ValidatingAdmissionPolicy %s failure, due to FailurePolicy=Ignore. Error: %w", definition.Name, err)
+			} else {
+				klog.Infof("ignored ValidatingAdmissionPolicy %s failure for binding %s, due to FailurePolicy=Ignore. Error: %w", definition.Name, binding.Name, err)
```

Comment:
for follow-up: don't dump ignored admission failures into the log

https://github.com/kubernetes/kubernetes/blob/1f9e20eb8617e312f9f61d28d93ed29e988d953d/staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/validating/dispatcher.go#L150-L172 is a good example of using metrics to record ignored errors

Reply:
Updated to not log the error. Adding metrics are tracked in [doc](https://docs.google.com/document/d/1rmVxzHL9tdmLPqD-QOXcdPxXTL7jvlxc5qFn4KI-yKI/edit) as follow-up